testcases/OpTestLPM.py

```diff
@@ -52,10 +52,12 @@ def setUp(self):
         self.src_mg_sys = self.cv_HMC.mg_system
         self.dest_mg_sys = self.cv_HMC.tgt_mg_system
         self.oslevel = None
-        self.src_lpar_vios = self.cv_HMC.lpar_vios.split(",")
-        self.dest_lpar_vios = conf.args.remote_lpar_vios.split(",")
         self.slot_num = None
         self.options = None
+        if conf.args.lpar_vios is not None:
+            self.src_lpar_vios = self.cv_HMC.lpar_vios.split(",")
+        if 'remote_lpar_vios' in conf.args:
+            self.dest_lpar_vios = conf.args.remote_lpar_vios.split(",")
```

Comment:
I was considering the fact that if the user specified either lpar_vios or remote_lpar_vios (instead of both), then we could check the MSP (future addition) of atleast the VIOS that the user has provided, instead of checking neither.

Reply:
it is must for user to give both vioses. as MSP must be checked on both vioses. we are assuming user is giving both vioses... if only one vios is given than ignore and continue with normal 

```
if conf.args.lpar_vios and 'remote_lpar_vios' in conf.args:
    initialize vios and split()
    for vios in all vioses:
        check MSP(vios)
```